### DISCUSSION TITLE
Where is documentation?

### DISCUSSION THREAD

**Discussion by beshur** [crowdin/crowdin-api-client-js#172] [General]
Hello.

I can't find the complete documentation for the library.

At least the method reference.

Thank you.

**Comment by DimaYashchyshyn-zz** [comment]
Hey @beshur 
Is this what you are looking for?
https://support.crowdin.com/api/v2/

**Reply by beshur** [reply]
Thank you, but I'm looking for the js library docs.

The readme only contains a couple of examples like:

```
import { ProjectsGroups } from '@crowdin/crowdin-api-client';

const projectsGroupsApi = new ProjectsGroups(credentials, {
  httpClientType: 'fetch'
});
```

**Reply by DimaYashchyshyn-zz** [reply]
Thanks for the clarification. We will work on it and create completed Library documentation. Still, it wouldn't be in an hour or so, that's why we would like to offer you a short screen-sharing call, so we can assist you with setting up as soon as possible. Kindly schedule the best time:
https://app.hubspot.com/meetings/crowdin-team/demo

**Reply by beshur** [reply]
Thank you! I'm just exporing for now, will book a time later.

**Reply by DimaYashchyshyn-zz** [reply]
Sure thing, take your time :) 
In case any assistance would be needed - just reach us at support@crowdin.com

**Reply by andrii-bodnar** [reply]
Hi @beshur, thanks a lot for your suggestion!

We have investigated this topic and decided to generate documentation using the [typedoc](https://typedoc.org/) tool with deploying to GH Pages.

[Enjoy](https://crowdin.github.io/crowdin-api-client-js/modules.html) :sparkles:

**Reply by beshur** [reply]
Wow, that fast! Thank you! 
👍
